tests/echoserver.yaml.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+: "${SECRETLESS_IMAGE:?"Need to set SECRETLESS_IMAGE non-empty, and available to cluster e.g. cyberark/secretless-broker:latest"}"
+
+# returns current namespace if available, otherwise returns 'default'
+current_namespace() {
+  cur_ctx="$(kubectl config current-context)" || exit_err "error getting current context"
+  ns="$(kubectl config view -o=jsonpath="{.contexts[?(@.name==\"${cur_ctx}\")].context.namespace}")" \
+     || exit_err "error getting current namespace"
+
+  if [[ -z "${ns}" ]]; then
+    echo "default"
+  else
+    echo "${ns}"
+  fi
+}
+
+cat << EOL
+kind: ClusterRole
```

Comment:
We can use a ClusterRole here to isolate RBAC to the test namespaces. This will help avoid flaky tests in case someone or a competing test might delete the ClusterRole